File: Web/ClientApp/src/models/experience.ts
import { SkillCategoryDetail } from "./skill";

export class Experience {
  public experienceId: number = 0;
  public name: string = '';
  public description: string = '';
  public start: Date;
  public end: Date;
  public SkillCategoryDetails: SkillCategoryDetail[];
}

File: DataAccess/Entities/Experience.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataAccess.Entities
{
    public class Experience
    {
        public int ExperienceId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime Start { get; set; }
        public DateTime? End { get; set; }
        public List<SkillCategoryDetail> SkillCategoryDetails { get; set; }
    }
}
